lightningd/peer_control.c

```diff
@@ -452,7 +452,7 @@ void add_peer(struct lightningd *ld, u64 unique_id,
 	peer->remote_funding_locked = false;
 	peer->scid = NULL;
 	peer->seed = NULL;
-	peer->balance = NULL;
+	peer->our_msatoshi = NULL;
```

Comment:
Why is this a pointer in the first place? It used to be an array for both sides, but tracking a single value we should just inline this.